.circleci/config.yml

```diff
@@ -99,9 +100,9 @@ workflows:
               ignore: /.*/
       - hmpps/helm_lint:
           name: helm_lint
-      - integration_tests:
-          requires:
-            - validate
+#      - integration_tests:
```

Comment:
As in the PR comment this is temporary want will be addressed as a separate piece of work.